### DISCUSSION TITLE
/dev/mapper/system-root filled

### DISCUSSION THREAD

**Discussion by Sascorr** [Security-Onion-Solutions/securityonion#8631] [Unsupported Versions] [answered]
Hello guys 

My SO standalone takes forever to start containers.  df -h indicates that dev/mapper/system-root is at 100% how do i clear this and stop if from happening again. overlay also indicates 100%

[bloom@bloomso ~]$ df -h
Filesystem               Size  Used Avail Use% Mounted on
devtmpfs                  31G     0   31G   0% /dev
tmpfs                     31G  536K   31G   1% /dev/shm
tmpfs                     31G  489M   31G   2% /run
tmpfs                     31G     0   31G   0% /sys/fs/cgroup
### /dev/mapper/system-root   98G   98G   20K 100% /
/dev/sda2                494M  188M  306M  39% /boot
/dev/mapper/system-nsm   192G   94G   99G  49% /nsm
/dev/mapper/system-tmp   2.0G   63M  1.9G   4% /tmp
overlay                   98G   98G   20K 100% /var/lib/docker/overlay2/59ae0a20ff2d7d29f8f65b89b46b93e6ed07efe657c3485ef6f86fb156e667cc/merged
overlay                   98G   98G   20K 100% /var/lib/docker/overlay2/3896c37695e6d02b36b5eb64da97678323d140d5f020e41aa28f2ad2ab57fb55/merged
tmpfs                    6.2G     0  6.2G   0% /run/user/1000
tmpfs                    6.2G     0  6.2G   0% /run/user/0
[bloom@bloomso ~]$ sudo grep log_size_limit /opt/so/saltstack/local/pillar/minions/*
[sudo] password for bloom:
grep: /opt/so/saltstack/local/pillar/minions/*: No such file or directory
[bloom@bloomso ~]$ ^C
[bloom@bloomso ~]$
[bloom@bloomso ~]$ sudo so-status
Cannot write to process directory. Do you have permissions to write to /var/cache/salt/minion/proc/20220830123314036148 ?

**Comment by dougburks** [comment, marked as answer]
You could use the following command to see what is taking up space:
```
sudo du --max-depth=1 -h /
```

However, a more productive use of your time would be to increase your disk space as you're barely above the minimum requirements. You'll want much more for a production deployment:
https://docs.securityonion.net/en/2.3/hardware.html#standalone-deployments

**Reply by Sascorr** [reply]
[bloom@bloomso ~]$ sudo du --max-depth=1 -h /
[sudo] password for bloom:
163M    /boot
0       /dev
94G     /nsm
du: cannot access ‘/proc/29816/task/29816/fd/4’: No such file or directory
du: cannot access ‘/proc/29816/task/29816/fdinfo/4’: No such file or directory
du: cannot access ‘/proc/29816/fd/3’: No such file or directory
du: cannot access ‘/proc/29816/fdinfo/3’: No such file or directory
0       /proc
2.1G    /run
0       /sys
30M     /tmp
37M     /etc
111M    /root
53G     /var
2.2G    /usr
99M     /home
0       /media
0       /mnt
42G     /opt
0       /srv
193G    /
it seems like /NSM /VAR and / are taking up my space however sudo so-nsm-clear and so-elastic-clear fails  whenever i try to run them

[bloom@bloomso ~]$ sudo so-elastic-clear
[sudo] password for bloom:

curl: (7) Failed connect to 10.2.1.8:9200; Connection refused


This script will delete all data (documents, indices, etc.) in the Elasticsearch database.

If you would like to proceed, please type AGREE and hit ENTER.

AGREE
Cannot write to process directory. Do you have permissions to write to /var/cache/salt/minion/proc/20220831113601124954 ?
Cannot write to process directory. Do you have permissions to write to /var/cache/salt/minion/proc/20220831113604668475 ?
Cannot write to process directory. Do you have permissions to write to /var/cache/salt/minion/proc/20220831113608323570 ?
=========================================================================
Stopping filebeat...
=========================================================================
Error response from daemon: No such container: so-filebeat
Error: No such container: so-filebeat
=========================================================================
Stopping logstash...
=========================================================================
Error response from daemon: No such container: so-logstash
Error: No such container: so-logstash
=========================================================================
Stopping elastalert...
=========================================================================
Error response from daemon: No such container: so-elastalert
Error: No such container: so-elastalert
Deleting data...
=========================================================================
Starting filebeat...

This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================

**Reply by dougburks** [reply]
You could investigate what's in `/var` but, as I mentioned in my previous reply, a more productive use of your time would be to increase your disk space as you're barely above the minimum requirements. You'll want much more for a production deployment:
https://docs.securityonion.net/en/2.3/hardware.html#standalone-deployments
